richard-danul/README.md

```diff
@@ -0,0 +1,25 @@
+# 02-jquery-dom
+
+**Author**: Richard & Danul
+**Version**: 1.0.0
+
+## Overview
+We built a blog app that dynamically shows articles on the page.
+
+## Getting Started
+1. Clone from github
+2. Navigate to richard-danul directory
+3. Open index.html
+
+## Architecture
+jQuery, HTML, Javascript, CSS.
+
+## Change Log
+09-05-2018 9:30am - Reviewed code

```

Comment:
make sure you are previewing your MD files, all "lists" need to have a `-` to make it a list or else markdown doesn't recognize that you want a line break so they are all appearing on one line, same with the credits section below.